packages/lascar/lascar.0.6/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis: "A library for manipulating Labeled Transition Systems in OCaml"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "https://github.com/jserot/lascar"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {build & >= "1.11"}
+  "ocamlfind" {>= "1.8"}
```

Comment:
Are you sure you require `ocamlfind`? I can't see any use of it by your dune build process

Reply:
It's used for compiling the examples (see example/Makefile.app).
But not for building the package itself.
I'll remove it.